CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Community Code of Conduct
+
+**Version 1.2  
+August 19, 2020**
```

Comment:
Update this, to be matching with this project.
"Version 1.0, July 5, 2022"

Reply:
Or is this is dedicated file from which we want to take the versioning (e.g., Eclipse Foundation template etc.)

Reply:
resolved by rephrasing to

This Code of Conduct adapted from the [Contributor Covenant](https://www.contributor-covenant.org) , version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct.html](https://www.contributor-covenant.org/version/1/4/code-of-conduct/) for the Eclipse Foundation Code of Conduct version 1.2, which also serves our project.